prow/plank/controller.go

```diff
@@ -180,6 +181,12 @@ func (c *Controller) syncJenkinsJob(pj kube.ProwJob) error {
 	if pj.Complete() {
 		return nil
 	} else if pj.Status.State == kube.TriggeredState {
+		if pj.Spec.MaxConcurrency > 0 && len(pjm[pj.Spec.Job]) >= pj.Spec.MaxConcurrency {
+			// do not start more jobs than allowed
+			// TODO(nclandolfi): log?
```

Comment:
Ok it looked like other parts of the codebase just use the default logrus logger? is that the right approach here?